pkg/ipamfetcher/customer.go

```diff
@@ -18,9 +18,17 @@ type customersResponse struct {
 }
 
 type customer struct {
-	CustomFields customFields `json:"Custom Fields"`
+	Contacts     []Contact    `json:"Contacts"`
+	CustomFields customFields `json:"custom_fields"`
 	ContactInfo  string       `json:"contact_info"`
 	ID           int          `json:"id"`
+	Name         string       `json:"name"`
```

Comment:
Task: Add this to the outgoing-gateway response validation too. Sorry, i overlooked this

Reply:
oh yeah.